app/controllers/pieces_controller.rb

```diff
@@ -27,7 +27,7 @@ def update
         redirect_to game_path(@game)  # redirect to game show page
       end
       format.json do
-        json_result = {valid: true, active: true, not_color: @color}
+        json_result = {valid: true, active: @piece.active, captured: @captured, not_color: @color}
```

Comment:
I don't believe this will work.  You aren't assigning the @captured variable in the controller.  Try to use

attr_accessor in the piece model

then when you are in the controller you can do @piece.captured to get the captured value.